README.adoc

```diff
@@ -250,11 +274,11 @@ mvn package
 Next, run the `docker build` commands to build container images for your application:
 [role='command']
 ```
-docker build -t system:1.0-SNAPSHOT system/.
-docker build -t inventory:1.0-SNAPSHOT inventory/.
+docker build --platform linux/amd64 -t system:1.0-SNAPSHOT system/.
+docker build --platform linux/amd64 -t inventory:1.0-SNAPSHOT inventory/.
 ```
 
-The `-t` flag in the `docker build` command allows the Docker image to be labeled (tagged) in the `name[:tag]` format. The tag for an image describes the specific image version. If the optional `[:tag]` tag is not specified, the `latest` tag is created by default.
+The `--platform` flag in the `docker build` command allows user to specify the target platform for the build output. As IBM Cloud Kubernetes service supports both `x86_64` and `s390x` architectures, the target platform will be `linux/amd64`. In addition, the `-t` flag in the command allows the Docker image to be labeled (tagged) in the `name[:tag]` format. The tag for an image describes the specific image version. If the optional `[:tag]` tag is not specified, the `latest` tag is created by default.
```

Comment:
```suggestion
The `--platform` flag in the `docker build` command specifies the target platform for the build output. Because IBM Cloud Kubernetes service supports both `x86_64` and `s390x` architectures, the target platform is `linux/amd64`. Also, the `-t` flag in the command allows the Docker image to be labeled (tagged) in the `name[:tag]` format. The tag for an image describes the specific image version. If the optional `[:tag]` tag is not specified, the `latest` tag is created by default.
```